shared/actions/fs-platform-specific.desktop.js

```diff
@@ -137,6 +137,7 @@ export const installKBFS = () => Saga.call(RPCTypes.installInstallKBFSRpcPromise
 export const installKBFSSuccess = (result: RPCTypes.InstallResult) =>
   Saga.sequentially([
     Saga.call(waitForMount, 0),
+    // $FlowFixMe @jzila please fix
```

Comment:
@jzila updated flow typing reveals this is incorrect

Reply:
Good catch, I'll fix in my refactor PR #11149